recipes/ruy/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["RUY_MINIMAL_BUILD"] = True
+        self._cmake.configure()
+        return self._cmake
+
+
+    def build(self):
+        # 1. Allow Shared builds
+        tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "ruy_cc_library.cmake"),
+                              "add_library(${_NAME} STATIC",
+                              "add_library(${_NAME}"
+                              )
+        # 2. Shared builds fail with undefined symbols without this. clog is a library from the cpuinfo
+        tools.replace_in_file(os.path.join(self._source_subfolder, "ruy", "CMakeLists.txt"),
+                              "set(ruy_6_cpuinfo \"cpuinfo\")",
+                              "set(ruy_6_cpuinfo \"cpuinfo;clog\")"
+                              )
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*.h", dst=os.path.join("include", "ruy"), src=os.path.join(self._source_subfolder, "ruy"))
+        self.copy(pattern="*", dst="lib", src="lib")
+        self.copy(pattern="*", dst="bin", src="bin")
+
+        tools.remove_files_by_mask(self.package_folder, "*.pdb")
+
+    def package_info(self):
+        self.cpp_info.libs = ["ruy_frontend",
+                            "ruy_context",
+                            "ruy_trmul",
+                            "ruy_thread_pool",
+                            "ruy_blocking_counter",
+                            "ruy_prepare_packed_matrices",
+                            "ruy_ctx",
+                            "ruy_allocator",
+                            "ruy_prepacked_cache",
+                            "ruy_tune",
+                            "ruy_wait",
+                            "ruy_apply_multiplier",
+                            "ruy_block_map",
+                            "ruy_context_get_ctx",
+                            "ruy_cpuinfo",
+                            "ruy_denormal",
+                            "ruy_have_built_path_for_avx",
+                            "ruy_have_built_path_for_avx2_fma",
+                            "ruy_have_built_path_for_avx512",
+                            "ruy_kernel_arm",
+                            "ruy_kernel_avx",
+                            "ruy_kernel_avx2_fma",
+                            "ruy_kernel_avx512",
+                            "ruy_pack_arm",
+                            "ruy_pack_avx",
+                            "ruy_pack_avx2_fma",
+                            "ruy_pack_avx512",
+                            "ruy_system_aligned_alloc",
+                            "ruy_profiler_instrumentation",
+                            "ruy_profiler_profiler"
+                            ]
```

Comment:
```suggestion
                            ]
        if self.settings.os in ["Linux", "FreeBSD"]:
            self.cpp_info.system_libs.append("pthread")
```

https://github.com/google/ruy/blob/9c56af3fce210a8a103eda19bd6f47c08a9e3d90/ruy/CMakeLists.txt#L111